zucchini/canvas.py

```diff
@@ -384,3 +384,81 @@ def set_submission_grade(self, course_id, assignment_id, user_id, grade,
 
         self._put('courses/{}/assignments/{}/submissions/{}'
                   .format(course_id, assignment_id, user_id), json)
+
+    def add_submission_comment(self, course_id, assignment_id, user_id,
+                               comment, files):
+        """
+        Adds a comment with optional file(s) attached to a submission
+
+        Info on uploading a file for comments. Warning! it's a very
+        involved process
+        https://canvas.instructure.com/doc/api/submission_comments.html
+
+        :param course_id: int canvas course id
+        :param assignment_id: int canvas assignment id
+        :param user_id: int canvas user_id
+        :param comment: string comment that will be added to submission
+        :param files: (optional) List of (paths to files, file mime type)
+        tuples that will be attached to comment
+        :return:
+
+        wow wouldn't it be great if you could auto detect the mime type?
+        yeah that would be great but windows doesn't have the binary
+        needed for python-magic to work
+        """
+
+        session = requests.Session()
+        session.headers = {'Authorization': 'Bearer %s' % self.token}
```

Comment:
might be a good idea to say `session.headers = self._headers()` so we don't have to repeat this

Reply:
oh yeah this is leftover from when I had some more headers